model/registry/geo.yaml

```diff
@@ -0,0 +1,113 @@
+            - id: 'an'
+              stability: experimental
+              value: 'AN'
+              brief: 'Antarctica'
+            - id: 'as'
+              stability: experimental
+              value: 'AS'
+              brief: 'Asia'
+            - id: 'eu'
+              stability: experimental
+              value: 'EU'
+              brief: 'Europe'
+            - id: 'na'
+              stability: experimental
+              value: 'NA'
+              brief: 'North America'
+            - id: 'oc'
+              stability: experimental
+              value: 'OC'
+              brief: 'Oceania'
+            - id: 'sa'
+              stability: experimental
+              value: 'SA'
+              brief: 'South America'
+      - id: continent.name
+        stability: experimental
+        type: string
+        brief: >
+          Name of the continent.
+        examples: [ 'North America', 'Europe' ]
+      - id: country.iso_code
+        stability: experimental
+        type: string
+        brief: >
+          Two-letter ISO Country Code ([ISO 3166-1 alpha2](https://en.wikipedia.org/wiki/ISO_3166-1#Codes)).
+        examples: [ 'CA' ]
+      - id: country.name
+        stability: experimental
+        type: string
+        brief: >
+          Country name.
+        examples: [ 'Canada' ]
+      - id: location.lon
+        stability: experimental
+        type: double
+        brief: >
+          Longitude of the geo location in [WGS84](https://en.wikipedia.org/wiki/World_Geodetic_System#WGS84).
+        examples: [ -73.614830 ]
+      - id: location.lat
+        stability: experimental
+        type: double
+        brief: >
+          Latitude of the geo location in [WGS84](https://en.wikipedia.org/wiki/World_Geodetic_System#WGS84).
+        examples: [ 45.505918 ]
+      - id: name
+        stability: experimental
+        type: string
+        brief: >
+          User-defined description of a location.
+        note: >
+          User-defined description of a location, at the level of granularity they care about.
+          Could be the name of their data centers, the floor number, if this describes a local physical entity, city names.
+          Not typically used in automated geolocation.
+        examples: [ 'boston-dc' ]
+      - id: postal_code
+        stability: experimental
+        type: string
+        brief: >
+          Postal code associated with the location.
+          Values appropriate for this field may also be known as a postcode or ZIP code and will vary widely from country to country.
+        examples: [ '94040' ]
+      - id: region.iso_code
+        stability: experimental
+        type: string
+        brief: >
+          Region ISO code ([ISO 3166-2](https://en.wikipedia.org/wiki/ISO_3166-2)).
+        examples: [ 'CA-QC' ]
+      - id: region.name
+        stability: experimental
+        type: string
+        brief: >
+          Region name.
+        examples: [ 'Quebec' ]
+      - id: timezone
```

Comment:
I'd expect timezone to be a thing on it's own. Someone can report timezone without geo context. E.g. when scheduling a meeting, the event that describes the meeting can have a timezone, but no physical location.

Reply:
There was a discussion on this in https://github.com/open-telemetry/semantic-conventions/pull/1116#discussion_r1662310967.

Let's handle it separately - I remove this attribute from the PR.